### ISSUE TITLE
New Bone Group - Skeleton?

### ISSUE THREAD

**Issue by clegarde** [spawaskar-cora/cora-docs#561]
Briefly discussed with @SachinPawaskarUNO from Data Summit, can we add a bone group that is the "Skeleton" that would include all elements with articulations and pair matches? Future enhancement, if possible.

**Comment by btrnew** [comment]
I don't know if anyone has brought this up yet ( @SachinPawaskarUNO @fedamann ), but one thing I have thought about in my dream of dreams is to construct the bone groups page visually like a clickable homunculus. Where upon entering the bone group page you're presented with the accession, provenance, and designator boxes and a blank homunculus: 

![CoRA homonculus example1](https://user-images.githubusercontent.com/43450732/68428541-7d55ae00-0150-11ea-82e2-4acd319e047f.PNG)

Then rather than removing elements from your "skeleton bone group" you click what you have available to you: 

![CoRA homonculus example](https://user-images.githubusercontent.com/43450732/68428652-ac6c1f80-0150-11ea-9cb6-77e492d8cbe0.PNG)

Then, like we have been discussing with the pelvic girdle, the associations between each of the elements you've presented are made automatically by the system rather than manually by the analyst. In the example above the sides of the bone would be assigned automatically, the automatic associations would be articulation between elements on each side, and like elements would be automatically pair matched. 

Things like trauma, pathology, and taphonomy are only going to cover an entire Arm and Shoulder, for example, in select situations. So when I am making a bone group it's unlikely I'm going to select something like antemortem sharp force trauma and osteomyelitis when making a bone group for the scapula, humerus, radius, ulna, and clavicle. Rather, I am more likely to make the bone group and then go in and edit the individual bones that the sharp force trauma directly pertains to. 

However, my second dream of dreams is to make bone group editing visual as well. So that I could tell the system "pull up all elements associated with X "skeleton bone group". The page would pull up the homunculus I made before, then I could click "Edit", and select the right humerus, left radius, and a right phalanx (for example). Now that the system knows which elements I am talking about I can input shared trauma, pathology, taphonomy, and/or completeness. Click save. Bam. Bone group edited. 

This dream edit screen would also be helpful if you're trying to pair match entire hands or feet from different bone groups because you could: 1. Pull up your homunculus, 2. Click all the elements you want to tell the system to automatically pair match, 3: Have a search function like we have on all the other screens to find any elements in another bone group, 4. Pick said element, and 5. It links the two bone groups under some unifying something and the next time you search your X "skeleton bone group" it pulls up the new elements you've pair matched or however you linked them. 

Anyways, it's all easier said then done, but that is my fantasy. 

TLDR: Can we do bone groups visually?
